src/curriculum/dataflow/dataflow.json

```diff
@@ -4,6 +4,24 @@
   "subtitle": "Computational Thinking in Biology",
   "pageTitle": "%unitTitle%",
   "demoProblemTitle": "%investigationTitle%",
+  "toolbar": [
+    {
+      "name": "select",
+      "title": "Select",
+      "iconId": "icon-select-tool"
+    },
+    {
+      "name": "text",
```

Comment:
When all is said and done, not sure if we'll even want the text tool in Dataflow, but it certainly makes sense for now so that we can show something.

Reply:
I'm also trying to reconcile how this tile model fits with a user creating programs and datasets and then opening them when needed.  Probably not relevant to this PR, but we're getting closer and closer to having to confront it.